public/app/features/provisioning/Shared/ProgressBar.tsx

```diff
@@ -5,9 +5,10 @@ import { useStyles2 } from '@grafana/ui';
 
 interface ProgressBarProps {
   progress?: number;
+  topBottomSpacing?: number;
 }
-const ProgressBar = ({ progress }: ProgressBarProps) => {
-  const styles = useStyles2(getStyles);
+const ProgressBar = ({ progress, topBottomSpacing }: ProgressBarProps) => {
+  const styles = useStyles2(getStyles, { topBottomSpacing });
```

Comment:
```suggestion
  const styles = useStyles2(getStyles, topBottomSpacing);
```

I don't think we can use objects here as the styles won't get memoized.